README.md

```diff
@@ -4,22 +4,30 @@
 
 </div>
 
-[![CircleCI](https://dl.circleci.com/status-badge/img/gh/graph-quilt/graphql-orchestrator-java/tree/master.svg?style=svg)](https://dl.circleci.com/status-badge/redirect/gh/graph-quilt/graphql-orchestrator-java/tree/master)
+A powerful Java library for aggregating and executing GraphQL operations from multiple microservices using a single unified schema.
 
 ![Master Build](https://github.com/graph-quilt/graphql-orchestrator-java/actions/workflows/main.yml/badge.svg)
+[Builds](https://circleci.com/gh/graph-quilt/graphql-orchestrator-java)
 
+### Introduction
 
+**graphql-orchestrator-java** simplifies the process of accessing data from various GraphQL microservices by providing a unified GraphQL schema. 
+This enables you to query multiple microservices through a single endpoint, reducing complexity and improving performance.
 
-[Builds](https://circleci.com/gh/graph-quilt/graphql-orchestrator-java)
+The library uses a [a recursive strategy](./mkdocs/docs/key-concepts/merging-types.md) to aggregate and combine the schemas from these microservices 
```

Comment:
We might need rephrase, like we have a two strategy for schema composition
1. recursive 
2. federation